packages/prover/README.md

```diff
@@ -37,13 +35,12 @@ console.log({balance, address});
 You can also invoke the package as binary.
 
 ```bash
-npm -i g @lodestar/prover
+npm i g @lodestar/prover
```

Comment:
This replaces the broken command with another broken command as this will now install the package [g](https://www.npmjs.com/package/g).

The intend here is to install the prover globally
```suggestion
npm i -g @lodestar/prover
```

Reply:
Good catch